### PR TITLE
added cross project toolkit workflow docs

### PR DESCRIPTION
**Pull request by bltasker**
first pass of some workflow docs to go along with cross project workflow feature.

### CODE REVIEW

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d69dbda9de462942dacfda38a2ee306b0ffe2442/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a7e1fb0331cb88072b95c101eafc21782f7ae469/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a7e1fb0331cb88072b95c101eafc21782f7ae469/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/73342439d27a2ce80c20633dcb8a5f13b488793a/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d514f57d872ad5bbd7b4f59dee8b4603a4917a46/index.html)

**Comment by bltasker** [comment]
Hi Shayna, this is my first developer doc, I think i've requested you to review the PR correctly. Did a few passes. I think it looks good from my side. Let me know if you have any concerns. Thanks

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+pagename: cross-project-workflow
+lang: en
+---
+# Cross Project Workflow
+
+## Best Practice
+
+Cross Project workflows in ShotGrid lets you re-use, share and work with assets across multiple projects. The most common method is to use a single "Asset Library" project. While we still recommend using a singular "Library" project to keep it simple, ShotGrid also allows you to link Assets across multiple active projects.
+
+There are 2 main ways we see studios re-using assets. Some studios may have flexibility and can re-use a source asset "as is". Other studios may have limitations or restrictions based on their projects and pipeline and will want to copy and asset over into a project and use it from there.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:13]:
```suggestion
There are two main ways to re-use Assets. The more nimble approach is to re-use a source asset "as is" (Direct Reference). However,  limitations or restrictions may be in place based on Projects and Pipelines—so copying the source Asset over into a project is also an option (Duplicating or Forking). 
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+---
+# Cross Project Workflow
+
+## Best Practice
+
+Cross Project workflows in ShotGrid lets you re-use, share and work with assets across multiple projects. The most common method is to use a single "Asset Library" project. While we still recommend using a singular "Library" project to keep it simple, ShotGrid also allows you to link Assets across multiple active projects.
+
+There are 2 main ways we see studios re-using assets. Some studios may have flexibility and can re-use a source asset "as is". Other studios may have limitations or restrictions based on their projects and pipeline and will want to copy and asset over into a project and use it from there. 
+
+* **Direct Reference** - Re-use a source asset as is. If that asset changes, you can keep your version or update to the latest version.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:15]:
```suggestion
* **Direct Reference** - Re-use a source Asset as is. If that asset changes, you can keep your Version or update to the latest Version.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* **Duplicating or Forking** - Import from another project (breaking any link) and build upon or take control and save to your current project.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:16]:
```suggestion
* **Duplicating or Forking** - Import the source Asset from another Project (breaking any connections to the source) to build on it or take control and save to your current Project.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+layout: default
+title: Cross Project Workflow
+pagename: cross-project-workflow
+lang: en
+---
+# Cross Project Workflow
+
+## Best Practice
+
+Cross Project workflows in ShotGrid lets you re-use, share and work with assets across multiple projects. The most common method is to use a single "Asset Library" project. While we still recommend using a singular "Library" project to keep it simple, ShotGrid also allows you to link Assets across multiple active projects.
```
[the remark this answers is not in the document]
Comment (shaynacohen) [reply docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:11]: @bltasker you'll want to make sure to replace all "ShotGrid" with the variable: {% include product %}

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+There are 2 main ways we see studios re-using assets. Some studios may have flexibility and can re-use a source asset "as is". Other studios may have limitations or restrictions based on their projects and pipeline and will want to copy and asset over into a project and use it from there. 
+
+* **Direct Reference** - Re-use a source asset as is. If that asset changes, you can keep your version or update to the latest version.
+* **Duplicating or Forking** - Import from another project (breaking any link) and build upon or take control and save to your current project.
+
+
+
+# Cross Project / Library Project Workflow
+This tutorial will show you how to use ShotGrid Toolkit to re-use your assets across multiple projects and build up an asset library thats easily accessible for artists in your studio. Let's take you through the 3 main steps to get this working.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:21]:
```suggestion
This tutorial will show you how to use {% include product %} Toolkit to re-use your Assets across multiple Projects and build up an Asset library that's easily accessible for Artists in your studio. Let's take you through the three main steps to get this up and running.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+1. **ShotGrid Assets linked across Projects** - using the "Linked Projects" linked field between Assets and Projects
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:23]:
```suggestion
1. **ShotGrid Assets linked across Projects** - use the **Linked Projects** field to link between Assets and Projects
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+2. **Loading/referencing assets into scenes** - using a dynamic tab in the Loader
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:24]:
```suggestion
2. **Loading/referencing Assets into Scenes** - use a dynamic tab in the Loader
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+3. **Keep Assets up to date** - using the Scene Breakdown App
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:25]:
```suggestion
3. **Keep Assets up to date** - use the Scene Breakdown App
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+
+## Step 1 - Linking Assets
+
+The first step of re-using assets is telling ShotGrid what other projects you would like to use an Asset in.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:30]:
```suggestion
The first step to re-use an Asset is by making {% include product %} aware of what other Projects you would like to use the source Asset in. 
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+1. **ShotGrid Assets linked across Projects** - using the "Linked Projects" linked field between Assets and Projects
+2. **Loading/referencing assets into scenes** - using a dynamic tab in the Loader
+3. **Keep Assets up to date** - using the Scene Breakdown App
+
+
+## Step 1 - Linking Assets
+
+The first step of re-using assets is telling ShotGrid what other projects you would like to use an Asset in. 
+
+From the source Project that has the original Asset your're looking to use, navigate to the Assets page.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:32]:
```suggestion
From the source Project that has the original Asset you're re-using, navigate to the Assets page.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+Make sure the "Linked Projects" field is visible on the page, if not you can add it. Then in that field add the project you want to use the asset in.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:33]:
```suggestion
Add the **Linked Projects** field to the Assets Page (if it's not already visible). Then, specify Project you want to re-use the Asset in.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+
+*See the detailed documentation* [here](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_site_configuration_ar_cross_project_asset_linking_html). 
+
+<img src="./images/cross-project/library-project-reuse.PNG" alt="Library Project Linked Projects Field"/>
+
+## Step 2 - Loading Assets
+With an Asset properly linked you can have a simple and dynamic tab in Loader to show your studios linked Assets. This helps artist and creative teams easily re-use assets across multiple projects.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:40]:
```suggestion
With an Asset properly linked via the **Linked Projects** field, you can have a simple and dynamic tab in Loader to show your studios linked, re-used Assets. This helps Artist and creative teams easily re-use Assets across multiple Projects. 
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+Make sure the "Linked Projects" field is visible on the page, if not you can add it. Then in that field add the project you want to use the asset in.
+
+*See the detailed documentation* [here](https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_site_configuration_ar_cross_project_asset_linking_html). 
+
+<img src="./images/cross-project/library-project-reuse.PNG" alt="Library Project Linked Projects Field"/>
+
+## Step 2 - Loading Assets
+With an Asset properly linked you can have a simple and dynamic tab in Loader to show your studios linked Assets. This helps artist and creative teams easily re-use assets across multiple projects. 
+
+Assuming the asset is properly linked, open your DCC of choice (Maya, Houdini, Alias, etc.), set the context by opening or saving your working scene. Once your scene context is correctly set, launch the Loader from the ShotGrid menu. Here you will see the tab called "Assets - Linked" allowing you to access Published Files for Assets across projects.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:42]:
```suggestion
Once the Asset is properly linked:
* Launch your content creation software of choice (Maya, Houdini, Alias, etc.)
* Set the context by opening or saving your working scene 
* Once your scene context is set, launch the Loader from the {% include product %} menu
* Now, you will see the tab **Assets - Linked** which allows you to access Published Files for Assets across Projects.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 14]
```diff
@@ -0,0 +1,67 @@
...
+
+
+<img src="./images/cross-project/loader-assets-linked.png" alt="Loader Tab for Assets - Linked"/>
+
+
+
+<img src="./images/cross-project/loader-assets-linked-description.png" alt="Loader Tab for Assets - Linked Description"/>
+
+
+From here, you can take advantage of your DCC Loader actions to do things like Import, if you plan to duplicate or fork your asset breaking any link to its source or Reference for an ongoing link to the current version and all subsequent versions of this asset.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:54]:
```suggestion
From here, you can take advantage of your content creation software Loader actions to do things like Import—if you plan to duplicate or fork your asset breaking any connections to its source—or Reference the Asset for an ongoing connection to the current Version and all subsequent Versions of this Asset.
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src="./images/cross-project/loader-actions.png" alt="Loader Actions"/>
+
+## Step 3 - Updating those assets
+If you choose to Reference this data and keep it pointing at a source somewhere else, you likely will want to see if the source changes. Maybe a different team is continuing to work on this asset and you need the latest version.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:59]:
```suggestion
If you choose to reference the Asset by keeping the connection to the source, you will likely want to see updates and changes to that source Asset. In this scenario, it's common that another team may continue to iterate on the source Asset and you will need the latest Version. 
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 16]
```diff
@@ -0,0 +1,67 @@
...
+
+
+From here, you can take advantage of your DCC Loader actions to do things like Import, if you plan to duplicate or fork your asset breaking any link to its source or Reference for an ongoing link to the current version and all subsequent versions of this asset.
+
+<img src="./images/cross-project/loader-actions.png" alt="Loader Actions"/>
+
+## Step 3 - Updating those assets
+If you choose to Reference this data and keep it pointing at a source somewhere else, you likely will want to see if the source changes. Maybe a different team is continuing to work on this asset and you need the latest version. 
+
+Using the Scene Breakdown 2 app in your config, you can see all the referenced items currently in your scene. If any of those references gets updated, the scene breakdown will inform you, share the published file notes about what changed, and allow you to update to the latest version or any other version of the published file.
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:61]:
```suggestion
Using the Scene Breakdown 2 App in your config will enable you to see all the referenced items currently in your Scene. When any of these references get updated, the Scene Breakdown will show you the conflicts, share the Published File notes about what changed, and allow you to update to the latest Version or any other Version of the Published File. 
```

File: docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md [hunk 17]
```diff
@@ -0,0 +1,67 @@
...
+<img src="./images/cross-project/loader-actions.png" alt="Loader Actions"/>
+
+## Step 3 - Updating those assets
+If you choose to Reference this data and keep it pointing at a source somewhere else, you likely will want to see if the source changes. Maybe a different team is continuing to work on this asset and you need the latest version. 
+
+Using the Scene Breakdown 2 app in your config, you can see all the referenced items currently in your scene. If any of those references gets updated, the scene breakdown will inform you, share the published file notes about what changed, and allow you to update to the latest version or any other version of the published file. 
+
+<img src="./images/cross-project/breakdown-cross-project.PNG" alt="Scene Breakdown 2"/>
+
+*Note: The updating of assets only apply's for Advanced Configs. If your studio uses a Basic config you will not have access to this app.*
```
Comment (shaynacohen) [inline docs/en/guides/pipeline-integrations/workflows/cross-project-workflow.md:65]:
```suggestion
*Note: The updating of Assets only applies when using an Advanced Configuration. If your studio uses a Basic Configuration, you will not have access to this Scene Breakdown App.*
```

**Review by shaynacohen — COMMENTED** [review]
Looks great, @bltasker ! I made some suggestions that should be easy for you to commit if you like them. Great work!

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/f26b5d71741b10aa28d601907686519cbb79891e/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/26ad39fbaaffc69e25667fcffead5aec59d277f0/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/ff2420961e20d3b11ff8bfa17d8a86e6e6740ea5/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/f694e80993f6b862ca205d77829adae60bd56754/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/3ab0410892d8c51d34f004c8d23df1bb24183766/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/a6e2e074c61b5bced645dc40f77d83818fe80696/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/62419ed40327f51421a350d91ffc1d6f8f05d172/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/088cac44d091551a9c5bd026e9d8ad91a7db4c7d/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/91fec633ebbe1346c73db2c0d23d89009aa5caa2/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/4157daf588bf7bf40b7c3d78f1625ab57d5ec4ab/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/7dd80c6d0ed3257aed50cc0c8115305d30d67bf4/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/c413b242f5ebd2d0af0d45cde69e9131fb95a959/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/3fa432e8f6fea3921a83001f6bcee5d2a5b0ed2b/index.html)

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/3fa432e8f6fea3921a83001f6bcee5d2a5b0ed2b/index.html)

**Comment by bltasker** [comment]
@shaynacohen thanks for the comments. Your feedback was great!
 I've committed all of this in and made a change to one image. This should be good to release now. Let me know what you think.

**Comment by toolkitty** [comment]
[Documentation Preview](http://sg-devdocs.s3-website-us-east-1.amazonaws.com/tk-doc-generator/d762a40e5c81d3689768af7a45a7443ca9fcd22d/index.html)

**Pull request merged**